### DISCUSSION TITLE
Video Game Module for Flipper Zero

### DISCUSSION THREAD

**Discussion by zelda-rageux** [slint-ui/slint#4649] [Ideas]
Hello everyone,
Do you think Slint can or could be used with this device?
https://blog.flipper.net/introducing-video-game-module-powered-by-raspberry-pi/
Have a nice day

**Comment by ogoffart** [comment]
I think so.
This seems to be about the same spec as the raspberry pi pico, which we support.

**Comment by zelda-rageux** [comment]
Excellent news, I will be reviewing the Video Game Module next week.
I will post a question in the FAQ section regarding licensing.
Thank you again for your answer.
